docs/ide/how-to-save-and-open-files-with-encoding.md

```diff
@@ -34,6 +34,8 @@ You can save files with specific character encoding to support bidirectional lan
 
      If you want to work with a file that you know is encoded in a specific way, you can tell Visual Studio to use that encoding when opening the file. The method you use depends on whether the file is part of your project.
 
+**NOTE:** if you want to save the project file with encoding, the **Save File As** option is not enabled until you unload the project.
```

Comment:
We have a specific formatting requirement for Notes in a topic. Please if you would, change your entry to the following, and we'll then merge your PR (which we really appreciate, by the way):

`> [!NOTE]`
`> If you want to save the project file with encoding, the **Save File As** option is not enabled until you unload the project.`

Reply:
Hi @TerryGLee thanks for the pointer! I have pushed another commit, hopefully I got it right (fingers crossed!).

Reply:
You did; thanks so much for your quick response! And again, we really appreciate your contribution. :)

Reply:
Thanks @TerryGLee have a great weekend ahead! :-)